test/dlc_tests/eks/mxnet/training/test_eks_mxnet_multinode_training.py

```diff
@@ -205,7 +205,10 @@ def _run_eks_multi_node_training_mpijob(namespace, job_name, remote_yaml_file_pa
     Function to run eks multinode training MPI job
     """
 
-    run(f"kubectl create namespace {namespace}")
+    does_namespace_exist = run(f"kubectl get namespace | grep {namespace}", warn=True)
+    
+    if not does_namespace_exist:
+        run(f"kubectl create namespace {namespace}")
```

Comment:
`run()` returns a [Result](http://docs.pyinvoke.org/en/stable/api/runners.html#invoke.runners.Result) object that will always have stdin, stdout, stderr, and other information on the object. Will `does_namespace_exist` ever be  `None` or empty?

Reply:
if the namespace does not exist, an empty string is returned and stored in `does_namespace_exist` variable followed by creation of new namespace.